src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -998,7 +998,7 @@
 examples:
   - name: List directories in a share. (autogenerated)
     text: |
-        az storage directory list --account-name MyAccount --share-name MyShare
+        az storage directory list --account-key 00000000 --account-name MyAccount --share-name MyShare
```

Comment:
actually the account key should be provided as parameters. In this way, the new one should be rejected.
I am not sure if the account key is manually added by myself.
May I know why Aladdin changes previous examples? Is there any replace mechanism? I would have expected appending examples.

Reply:
You can check git history whether it's added by yourself.

Reply:
Hi, Roshanak @albaloo, could you please answer this?